### DISCUSSION TITLE
Does the latest firmware allow Motion Detection to be toggled on / off ?

### DISCUSSION THREAD

**Discussion by steve1011011** [roleoroleo/yi-hack-MStar#452] [Q&A]
I have a few devices with an old 0.2.5 version of Yi-Hack for model suffix y203c .  I see no way of toggling motion detection on or off. You have to sign into the Yi app for that.

I want to remove my cloud dependency on these devices completely.

Does the latest firmware version have this option?

I stuck these in the cupboard a few years back, as they didn't do what I wanted . Looks like things have come on. 

Is there some sort of limitation that forces the App to be used to toggle motion detection?

It's not a massive issue. I could just turn it on always, and use automation in Home Assistant to ignore MTQQ topics until certain times. 

EDIT

I went ahead and upgraded to 0.4.7 anyway as I wanted to avail of the HA integration via HACs. That answered my question :)

"Motion Detection | You can't enable/disable motion detection here, please use mobile app."

So more of a question......why is this the case ? Is motion detection tied to the cloud? If I turn on motion detection, will it permanently be on even if Yi/Kami cloud turned off their cloud services?

**Comment by roleoroleo** [comment]
> So more of a question......why is this the case ? Is motion detection tied to the cloud? If I turn on motion detection, will it permanently be on even if Yi/Kami cloud turned off their cloud services?

I don't know exactly what happens when you turn on/off motion detection with the app.
But the cam doesn't receive any message from the cloud, so I don't know how to clone this command.

**Comment by olinorwell** [comment]
I have all the cloud features disabled and the camera can't access the Internet.

Motion detection still works fine. So can confirm zero connection to an app/cloud. I would use the app to turn it on or off, then you can dispose of the app.
